packages/mui-joy/src/Button/Button.tsx

```diff
@@ -127,10 +127,11 @@ export const getButtonStyles = ({
         '--Button-gap': '0.75rem',
         minHeight: 'var(--Button-minHeight, 2.75rem)',
         fontSize: theme.vars.fontSize.md,
-        paddingBlock: '0.375rem',
+        paddingBlock: '0.5rem',
         paddingInline: '1.5rem',
       }),
       WebkitTapHighlightColor: 'transparent',
+      boxSizing: 'border-box',
```

Comment:
Need to explicitly add this, otherwise, there will be a lot of visual regression changes due to padding/line-height changes.

Material UI also explicitly sets this value.